packages/chain-adapters/src/cosmossdk/cosmos/CosmosChainAdapter.ts

```diff
@@ -381,7 +377,7 @@ export class ChainAdapter extends CosmosSdkBaseAdapter<KnownChainIds.CosmosMainn
       const txToSign: CosmosSignTx = {
         addressNList,
         tx: utx,
-        chain_id: CHAIN_REFERENCE.CosmosHubMainnet,
+        chain_id: this.chainId,
```

Comment:
regression

Reply:
addressed in https://github.com/shapeshift/lib/pull/914/commits/080c21f5f8f5aa8c96b55e31126c9b828da30d52